addons/web/static/tests/views/fields/image_field.test.js

```diff
@@ -761,3 +761,42 @@ test("unique in url does not change on record change if reload option is set to
     await clickSave();
     expect(getUnique(queryFirst(".o_field_image img"))).toBe("1659688620000");
 });
+
+test("convert image to webp", async () => {
+    onRpc(({ method, model, args }) => {
+        if(method == "create_unique" && model == "ir.attachment") {
```

Comment:
this code hasn't been linted. You can enable this via ./addons/web/tooling/enable.sh (a pre-commit hook will then lint/prettify your diff)